src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -278,16 +278,20 @@ def _flexible_server_params(command_group):
 
         with self.argument_context('{} flexible-server delete'.format(command_group)) as c:
             c.argument('server_name', id_part='name', options_list=['--name', '-n'], arg_type=server_name_getter_arg_type)
-            c.argument('force', options_list=['--force'], action='store_true',
-                       help='Delete the server without prompt')
+            c.argument('yes', options_list=['--yes', '-y'], action='store_true',
+                       help='Do not prompt for confirmation.')
 
         with self.argument_context('{} flexible-server restore'.format(command_group)) as c:
             c.argument('server_name', options_list=['--name', '-n'], arg_type=overriding_none_arg_type,
                        help='The name of the new server that is created by the restore command.')
-            c.argument('source_server', options_list=['--source-server'],
-                       help='The name or resource ID of the source server to restore from.')
-            c.argument('restore_point_in_time', options_list=['--time'],
+            c.argument('restore_point_in_time', options_list=['--restore-time'],
```

Comment:
same as above.

Reply:
PMs decided to go with restore-time. Originally it was --restore-point-in-time but we needed to fix this due to linter issue and we are going to stick to this parameter name from now on..